### PR TITLE
fix #79016 openreach.com

### LINKED ISSUE #79016: www.openreach.com
### Issue URL (Annoyance)
 [https://www.openreach.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.openreach.com%2F)
### Comment
Username: @gigabit2
> Cookies policy should be blocked with Annoyances Filter turned on.
> 
> Also there is this captcha thing at the bottom right, perhaps this should also be blocked.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v1.8.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist

### PR DESCRIPTION
**Pull request by wout-codes**
#79016

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -5396,6 +5396,8 @@ cmp.computerbild.de,consent.giga.de,cmp.fitbook.de,consent.spieletipps.de,consen
...
 privacy.madame.de#%#AG_onLoad(function(){if(!document.cookie.includes("euconsent-v2")){var g=new MutationObserver(function(){setTimeout(function(){var b=document.querySelector('button.no-children[path="\[0,2,1\]"]');b&&b.click();},300);setTimeout(function(){var c=document.querySelector('div[title="Berechtigte Interessen"]'); c&&c.click();},300);setTimeout(function(){var d=document.querySelectorAll('label[aria-checked="true"] > .off'); d.forEach(function(d){d.click();});},800); var f=document.querySelector('button.no-children[path="\[0,3,0\]"]'); setTimeout(function(){f&&f.click();},1000);});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},3E4)}});
 !
+openreach.com#%#//scriptlet("set-cookie", "NecessaryCookie", "true")
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:5399]: Is something broken if popup is hidden?

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:5399]: Yeah, the live chats.

**Review by wout-codes — COMMENTED** [review]
-

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:5399]: @Alex-302

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:5399]: Cannot find a chat(without AG)

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:5399]: If I am not mistaken, it requires an account.

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:5399]: I don't see register/login buttons)

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:5399]: Same I guess I was wrong)

**Pull request closed without merging**